pre_election_sample_feed.xml

```diff
@@ -0,0 +1,1841 @@
+<?xml version="1.0"?>
+<ElectionReport xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <Election>
+    <BallotStyleCollection>
+      <!-- The various ballot styles in use by the different precincts. -->
+      <BallotStyle objectId="bs00000">
+        <GpUnitIds>ru_pre00000 ru_pre99999 ru_pre90139 ru_pre90666 ru_pre91781sp0001 ru_pre92175</GpUnitIds>
```

Comment:
Shall we "ru" instead of "ru_" for consistency? Prefixes of other elements do not have the "_".